### DISCUSSION TITLE
bisheng是否支持接口暴露

### DISCUSSION THREAD

**Discussion by stereoF** [dataelement/bisheng#92] [General]
用bisheng创建应用后，是否支持通过web接口访问这个应用，以集成在自己的web服务中，而不是使用bisheng自己的页面进行使用？

**Comment by wisdomqin** [comment]
支持的，可查看这个在线文档 https://dataelem.feishu.cn/wiki/CpXNw6ZTjiKimjkv66Wcw0EWnHc

**Reply by ieayoio** [reply]
私有化部署支持吗，我测试搭建了一个，发现没有对外发布的功能：
![image](https://github.com/user-attachments/assets/4e98dc3e-baca-4489-9712-c7aa92014212)

**Reply by ieayoio** [reply]
> 私有化部署支持吗，我测试搭建了一个，发现没有对外发布的功能： ![image](https://private-user-images.githubusercontent.com/7099604/400275799-4e98dc3e-baca-4489-9712-c7aa92014212.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzcwMjY1MTIsIm5iZiI6MTczNzAyNjIxMiwicGF0aCI6Ii83MDk5NjA0LzQwMDI3NTc5OS00ZTk4ZGMzZS1iYWNhLTQ0ODktOTcxMi1jN2FhOTIwMTQyMTIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDExNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMTZUMTExNjUyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9M2YyYmRiMDE3NDQ0NTI3MjgxMmZmYjVkYzVhYTA3MDZjNjk5ZjQ1NTI5M2E2YThjOWU1MjU0NmJhOTcxY2E3MiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.e_s8aZRUZ17KOe0JAAan5_eIehhKzd7TM4AwRndqfAA)
升级版本后已经支持了
